WordPress/Classes/ViewRelated/Menus/MenuItemEditingViewController.m

```diff
@@ -0,0 +1,470 @@
+        self.typeViewController = segue.destinationViewController;
+    }
+}
+
+- (BOOL)prefersStatusBarHidden
+{
+    [self.headerView setNeedsTopConstraintsUpdateForStatusBarAppearence:self.headerView.hidden];
+    return self.headerView.hidden;
+}
+
+- (void)traitCollectionDidChange:(UITraitCollection *)previousTraitCollection
+{
+    [super traitCollectionDidChange:previousTraitCollection];
+    [self updateLayoutIfNeeded];
+}
+
+- (void)loadContentLayoutConstraints
+{
+    {
+        self.layoutConstraintsForDisplayingTypeView = @[
+                                                        [self.typeViewController.view.leadingAnchor constraintEqualToAnchor:self.contentView.leadingAnchor],
+                                                        [self.typeViewController.view.trailingAnchor constraintEqualToAnchor:self.contentView.trailingAnchor],
+                                                        [self.sourceViewController.view.leadingAnchor constraintEqualToAnchor:self.typeViewController.view.trailingAnchor],
+                                                        [self.sourceViewController.view.widthAnchor constraintEqualToAnchor:self.contentView.widthAnchor]
+                                                        ];
+    }
+    {
+        self.layoutConstraintsForDisplayingSourceView = @[
+                                                          [self.typeViewController.view.trailingAnchor constraintEqualToAnchor:self.contentView.leadingAnchor],
+                                                          [self.typeViewController.view.widthAnchor constraintEqualToAnchor:self.contentView.widthAnchor],
+                                                          [self.sourceViewController.view.leadingAnchor constraintEqualToAnchor:self.contentView.leadingAnchor],
+                                                          [self.sourceViewController.view.trailingAnchor constraintEqualToAnchor:self.contentView.trailingAnchor]
+                                                          ];
+    }
+    {
+        self.layoutConstraintsForDisplayingSourceAndTypeViews = @[
+                                                                  [self.typeViewController.view.leadingAnchor constraintEqualToAnchor:self.contentView.leadingAnchor],
+                                                                  [self.typeViewController.view.widthAnchor constraintEqualToConstant:180.0],
+                                                                  [self.sourceViewController.view.leadingAnchor constraintEqualToAnchor:self.typeViewController.view.trailingAnchor],
+                                                                  [self.sourceViewController.view.trailingAnchor constraintEqualToAnchor:self.contentView.trailingAnchor]
+                                                                  ];
+    }
+}
+
+- (BOOL)shouldLayoutForCompactWidth
+{
+    if (IS_IPAD) {
```

Comment:
Not sure this check is needed. Layout on the iPad should never be horizontally compact.  Regardless, we're want to avoid the `IS_IPAD` macro in favor of the `WPDeviceIdentification` methods recently added to the shared pod.